cldf/requirements.txt

```diff
@@ -0,0 +1,62 @@
+appdirs==1.4.3
```

Comment:
A nice side-effect of cldfbench is that we save the python environment - for makecldf as well as for the commands.